src/sth_app.js

```diff
@@ -24,7 +24,9 @@
     }
 
     if (err) {
-      sthLogger.error(err);
+      sthLogger.fatal(err.toString(), {
+        operationType: sthConfig.OPERATION_TYPE.SHUTDOWN
```

Comment:
Just to understand it, the `operationType` is passed as useful information for the logging process, right? I mean, in this particular case something like "a fatal error occured when trying ti shutdown the server" is logged.